scripts/skills/actives/rf_pocket_sand_skill.nut

```diff
@@ -0,0 +1,87 @@
+this.rf_pocket_sand_skill <- this.inherit("scripts/skills/actives/throw_dirt_skill", {
+	m = {
+		RemainingUses = 0
+	},
+
+	function create()
+	{
+		this.throw_dirt_skill.create();
+
+		this.m.ID = "actives.rf_pocket_sand_skill";
+		this.m.Name = "Throw Pocket Sand";
```

Comment:
I'd say, in a separate commit, also add this skill to the `PossibleSkills` of the same ai behavior that uses throw_dirt. And then add that here as `AIBehaviorID`. This way this skill can be given to enemies as well.